jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -536,32 +536,32 @@ properties:
             fits_keyword: OPMODE
             blend_table: True
           gwa_xtilt:
-            title: Grating Y tilt angle relative to mirror
+            title: Grating wheel tilt along instrmnt model Y axis
             type: number
             fits_keyword: GWA_XTIL
             blend_table: True
           gwa_ytilt:
-            title: Grating X tilt angle relative to mirror
+            title: Grating wheel tilt along instrmnt model X axis
```

Comment:
Spelling of instrmnt: Is there a reason for the abbreviation?

Reply:
See above